custom/icds_reports/static/js/directives/month-filter/month-filter.directive.js

```diff
@@ -7,17 +7,11 @@ function MonthModalController($location, $uibModalInstance, dateHelperService) {
     vm.years = [];
     vm.monthsCopy = [];
     vm.showMessage = false;
-    var reportStartDates = {
-        'sdd': new Date(2019, 1),
-    };
+    var reportStartDates = dateHelperService.getReportStartDates();
```

Comment:
what's this variable used for now?

Reply:
its used to decide whether to show the alert (SDD available only from feb-2019)

Reply:
but doesn't all that happen in the service? is it referenced in this directive?

Reply:
https://github.com/dimagi/commcare-hq/blob/0e1658ca342d6ddccfc7585aae55b76c01f535cb/custom/icds_reports/static/js/directives/month-filter/month-filter.directive.js#L45

In this case its happening in directive

Reply:
ah, thanks. missed that